docs/topics/lombok.md

```diff
@@ -0,0 +1,149 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+Since 1.5.20, Kotlin supports the experimental Lombok Kotlin compiler plugin. This plugin allows the use of
+Java's Lombok-generated declarations in mixed Kotlin/Java sources.
+If Java's Lombok-generated declarations are in another module than the Kotlin code that uses these declarations, 
+then you don't need to use this plugin.
+
+[Lombok](https://projectlombok.org/) does Java code generation itself, not the Lombok Kotlin compiler plugin. 
+So, you still need to configure Lombok as usual, when using this plugin. 
+Learn more about [how to make the plugin seeing Lombok's config](#defining-the-place-of-lombok-config).
+
+## Support of annotations
+
+Learn which annotations the plugin supports and the
+[current state of Lombok Kotlin compiler plugin](https://github.com/JetBrains/kotlin/blob/master/plugins/lombok/lombok-compiler-plugin/README.md).
+
+> Generation of `@Builder` annotation won't be supported in the current prototype. However, you can give your feedback
```

Comment:
Not sure this is a good place for this note. We don't say a word about what is supported but explicitly say "We don't support Builder!" - this can cause negative perception.

Reply:
Ah, I've skipped this comment by a mistake, sorry. In the DM we decided to turn this note just to the text: "Currently, we don't have plans on supporting the @Builder annotation. However, we can consider this if you vote
for [`@Builder` in YouTrack](https://youtrack.jetbrains.com/issue/KT-46959)."

> We don't say a word about what is supported
We do this because the source of truth is the README under the link, and there would be redundant work to keep our doc synced with this README.

If to leave this heading as it's now: "Support of annotations", it seems to me it's ok to say here about @Builder. Otherwise, I don't know where to say about this but not in annotations, it can be negatively perceived everywhere.

Do you have any ideas where to put it?